atom/browser/importer/profile_writer.cc

```diff
@@ -26,7 +28,13 @@
 
 namespace importer {
 void ShowImportLockDialog(gfx::NativeWindow parent,
-                          const base::Callback<void(bool)>& callback) {}
+                          const base::Callback<void(bool)>& callback) {
```

Comment:
because the current behavior is requiring users to do importing again (importer process ends)